src/main/java/by/exadel/internship/entity/Internship.java

```diff
@@ -78,11 +89,12 @@
             inverseJoinColumns = @JoinColumn(name = "ui_u_id"))
     private List<User> users;
 
-    @ManyToMany
+    @ManyToMany(cascade = CascadeType.ALL)
```

Comment:
Do you really need cascade type all here? It will affect all operations with changes and entity removing

Reply:
i added , merge instead this one